src/main/java/com/kodilla/ecommercee/entity/Cart.java

```diff
@@ -25,7 +25,7 @@
     @JoinColumn(name = "cart_id")
     private List<Item> items = new ArrayList<>();
 
-    private boolean closed = false;
+    private boolean closed = true;
```

Comment:
To niepotrzebne. Typy prymitywne boolean inicjowane są false. Jak zakładasz nowy Cart, to od razu go zamykasz -> closed=true. To trzeba zrobić na etapie zakładania nowego zamówienia na podstawie istniejącego cart. W CartService w metodzie createAnOrder tuz przed zapisem (return ...) dodaj cart.setClosed(true) i po sprawie.